docs/pages/admin-guides/teleport-policy/integrations/azure-sync.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Discover Azure Access Patterns with Teleport Policy
+description: Describes how to import and visualize Azure subscription access patterns using Teleport Policy and Access Graph.
+---
+
+Teleport Policy streamlines and centralizes access management across your entire infrastructure. You can view access
+relationships in seconds, viewing unified, up-to-date relationships and policies between all users, groups, and
+computing resources.
+
+Teleport Policy with Access Graph offers insights into access patterns within your Azure subscription. By scanning
+users, groups, service principals, and role definitions, it provides a visual representation and aids in enhancing the permission model
+within your Azure environment. This functionality enables you to address queries such as:
+
+- What resources are accessible to Azure users and roles?
+- Which groups belong to other groups and how does that impact resource access?
+- How broad or narrow is scoping across the subscription when assigning roles to users and groups?
+
+Utilizing the Access Graph to analyze permissions within an Azure subscription necessitates the setup of the Access
+Graph (AG) service, a Discovery Service, and integration with your Azure subscription.
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product that is available to Teleport Enterprise customers.
+
+After logging in to the Teleport UI, go to the Management tab. If enabled, Access Graph options can be found under
+the Permission Management section.
```

Comment:
The Teleport 17 UI doesn't have a Management tab.

We might get a higher quality piece of documentation if we pulled up the UI and went through the feature writing the guide from scratch rather than copying from an existing guide. When we just copy paste from other guides we also copy old and outdated info and create more work for ourselves to fix.